### ISSUE TITLE
ValueError: Failed to start Kaleido subprocess. Error stream: [NONE]

### ISSUE THREAD

**Issue by marianoju** [plotly/Kaleido#164]
Running Python 3.8.17 in a conda environment on Windows 10. I have created a dedicated virtual environment and installed `plotly` and `python-kaleido`. I have then executed the following script: 

```
import plotly 

fig = plotly.graph_objects.Figure()

fig.write_image("plot.png")
```

Output is as follows: 

```
Traceback (most recent call last):
  File "write_image.py", line 5, in <module>
    fig.write_image("plot.png")
  File "C:\Users\marianoju\anaconda3\envs\plotly\lib\site-packages\plotly\basedatatypes.py", line 3840, in write_image
    return pio.write_image(self, *args, **kwargs)
  File "C:\Users\marianoju\anaconda3\envs\plotly\lib\site-packages\plotly\io\_kaleido.py", line 267, in write_image
    img_data = to_image(
  File "C:\Users\marianoju\anaconda3\envs\plotly\lib\site-packages\plotly\io\_kaleido.py", line 144, in to_image
    img_bytes = scope.transform(
  File "C:\Users\marianoju\anaconda3\envs\plotly\lib\site-packages\kaleido\scopes\plotly.py", line 153, in transform
    response = self._perform_transform(
  File "C:\Users\marianoju\anaconda3\envs\plotly\lib\site-packages\kaleido\scopes\base.py", line 293, in _perform_transform
    self._ensure_kaleido()
  File "C:\Users\marianoju\anaconda3\envs\plotly\lib\site-packages\kaleido\scopes\base.py", line 198, in _ensure_kaleido
    raise ValueError(message)
ValueError: Failed to start Kaleido subprocess. Error stream:


```

The empty Error stream makes this error particularly difficult to debug. I have tried installing 
- `plotly` 5.9.0 from defaults and `python-kaleido` 0.2.1 from conda-forge 
- `plotly` 5.11.0 and `python-kaleido` 0.2.1 from conda-forge 
- `plotly` 5.15.0 and `python-kaleido` 0.0.3 from plotly 
- `plotly` 5.15.0 from plotly and `python-kaleido` 0.2.1 from conda-forge 
- `plotly` 5.15.0 and `python-kaleido` 0.2.1 from conda-forge 

In short, I have not found a way to fix the error. I would be very grateful for any help.

**Comment by milesgpt** [comment]
my issue with windows 10 in conda was the spaces in the Program Files path throwing it off.

reverting to this older version worked from me for my windows 10 env

`pip install --upgrade "kaleido==0.1.*"`

**Comment by etiennedemontalivet** [comment]
Same issue, fixed by reverting to version `0.1.0.post1`
``` bash
pip install "kaleido==0.1.0.post1"
```

**Comment by etiennedemontalivet** [comment]
Issue was fixed with version `0.1.0.post1` on **Windows**.
Now this simple code :

**Comment by gvwilson** [comment]
Thanks for your interest in Kaleido. We are currently working on an overhaul that might address your issue - we hope to have news in a few weeks and will post an update then. Thanks - @gvwilson

**Comment by rhiannaleslie1** [comment]
Hi there, I am having the same error and downgrading to `0.1.0.post1` didn't solve my issue. I'm working with windows 10, I wasn't getting this bug last week on the same laptop. Are there any updates on a fix? Thanks

**Comment by konosubakonoakua** [comment]
same here...

**Comment by ayjayt** [comment]
Hi @rhiannaleslie1 an @konosubakonoakua - there should be a new version of kaleido out today. It needs chromium installed separately. 

If you'd like help, please post an issue with your output, and its better if you set env var `KALEIDO_DEBUG=true` before running it so I can see more output.

Some people are pinning backwards, but we can't support those older versions. Thanks so much!

**Comment by ayjayt** [comment]
> > Hi [@rhiannaleslie1](https://github.com/rhiannaleslie1) an [@konosubakonoakua](https://github.com/konosubakonoakua) - there should be a new version of kaleido out today. It needs chromium installed separately.
> > If you'd like help, please post an issue with your output, and its better if you set env var `KALEIDO_DEBUG=true` before running it so I can see more output.
> > Some people are pinning backwards, but we can't support those older versions. Thanks so much!
> 
> Hi [@ayjayt](https://github.com/ayjayt) , Can I use edge?
@konosubakonoakua 
Other users have had success with edge if they set the env var `BROWSER_PATH` to its path
